.github/workflows/go.yml

```diff
@@ -89,13 +89,14 @@ jobs:
     strategy:
       matrix:
         test-setups:
-        - {name: "Multinode",                shortname: "multinode",    ext: true,  tsdb: true,  tsdb2: true,  tsdboss: false, multi: true,  pg: 13}
-        - {name: "W/O Promscale Extension",  shortname: "wo-prom-ext",  ext: false, tsdb: true,  tsdb2: true,  tsdboss: false, multi: false, pg: 13}
-        - {name: "Plain Postgres",           shortname: "plain-pg",     ext: false, tsdb: false, tsdb2: false, tsdboss: false, multi: false, pg: 13}
         - {name: "Timescaledb 1.x (pg 12)",  shortname: "tsdb1x-pg-12", ext: true,  tsdb: true,  tsdb2: false, tsdboss: false, multi: false, pg: 12}
         - {name: "Plain Postgres (12)",      shortname: "plain-pg-12",  ext: false, tsdb: false, tsdb2: false, tsdboss: false, multi: false, pg: 12}
-        - {name: "TimescaleDB-OSS",          shortname: "tsdb-oss",     ext: false, tsdb: true,  tsdb2: true,  tsdboss: true,  multi: false, pg: 13}
-        - {name: "pg 14",                    shortname: "pg14",         ext: true,  tsdb: true,  tsdb2: true,  tsdboss: false, multi: false, pg: 14}
+        - {name: "Plain Postgres (13)",      shortname: "plain-pg-13",  ext: false, tsdb: false, tsdb2: false, tsdboss: false, multi: false, pg: 13}
+        - {name: "Multinode",                shortname: "multinode",    ext: true,  tsdb: true,  tsdb2: true,  tsdboss: false, multi: true,  pg: 14}
```

Comment:
I think we are missing the canonical single node for pg 14 which used to be called "pg 14". and is `ext: true, tsdb: true, tsdb2: true, tsdboss: false, multi: false, pg: 14`

Reply:
I have added this back in.